components/explore-the-data-page/DatasetDetails.js

```diff
@@ -18,6 +19,9 @@ export default function DatasetDetails(props) {
         </ul>
       </div>
       <div className="col-right">
+        <Link href="/map">
+          <a className="btn btn--primary">View Map</a>
```

Comment:
When I saw this button in the same box as the download CSV button and the total count, I assumed the map would also respect the selected filters and dataset. What do you think of moving this out of this box?

Reply:
I agree, this is bothering me as well and I wish we could use query strings here to select the map we want.

I'm not really sure where else to put the map link, though I definitely agree it's misleading where it is now. Do you have any ideas?

P.s. this is really a tricky UI to build!!

Reply:
Agreed, it's definitely a tricky UI to work with. There's already a lot going on.

Maybe not the most exciting, but I think a simple link toward the top might make it seem less connected to the filtered data.

<img width="1191" alt="Screen Shot 2019-12-16 at 8 12 05 PM" src="https://user-images.githubusercontent.com/7942714/70964382-82a3f380-2040-11ea-9354-708cf380ed4f.png">

Reply:
Unrelated, but that red text has a `mobile-only` class that doesn't seem to be defined for that element. I wonder if we should hide it on larger screens?

Reply:
Yeah I noticed this...at one point that class was defined, I'm not sure what happened. Anyhow, I've reinstated it.

Reply:
Awesome, looks great!